### ISSUE TITLE
why does it need to be on a seperate server?

### ISSUE THREAD

**Issue by ghost** [stephen-mc-pl/OneInTheQuiver#3]
it cancels block break/place and other stuff in the whole server which makes it only good when its on a seperate server, why??

can't u just make it like any other game ?

**Comment by buchwasa** [comment]
Because that's how I design my games. I don't design them to be run all in one server because I'd have to deal with everyone complaining about that instead, I'll leave it as it is, I might change it in the future.

**Comment by buchwasa** [comment]
Hello there, I've recently picked up on this project again, and this is presently being worked on, sorry for the very long time this has taken.

**Comment by ghost** [comment]
unfortunately i cannot use it now since you updated it to api 4 while my server is still at api 3, perhaps you can make an api3 branch and make the game able to be used in a server along with other games? only in this case i can use it

**Comment by buchwasa** [comment]
I will admit that I originally did not plan to put API 4 on master branch, it was a mistake, but PM4 will hopefully be coming out before MC 1.17, so the plugin will just be ready for that. 

tl;dr - PM3 support is something I probably won't be going back to.

**Comment by ElectroGamesDev** [comment]
WHy wont you use API 3? API 4 isnt even out yet meaning the plugin is useless lol

**Comment by buchwasa** [comment]
All of my plugins are API 4 ready, as I said, PM4 expected to release before 1.17. With this, I didn't mean to use API 4 on master branch, I made a mistake, but decided not to revert it.
